config/environments/development.rb

```diff
@@ -22,9 +22,12 @@
       'Cache-Control' => "public, max-age=#{2.days.to_i}"
     }
   else
-    config.action_controller.perform_caching = false
+    config.action_controller.perform_caching = true
 
-    config.cache_store = :null_store
+    config.cache_store = :memory_store
+    config.public_file_server.headers = {
+      'Cache-Control' => "public, max-age#{2.days.seconds.to_i}"
```

Comment:
This looks good. We might want to circle back to Ian's thoughts on the 2 day caching limit at some point

Reply:
I agree with that and setting a "limit" to different attributes such as title vs reviews